tools/run_tests/python_tests.json

```diff
@@ -13,6 +13,8 @@
   "grpc.framework.face.event_invocation_synchronous_event_service_test",
   "grpc.framework.face.future_invocation_asynchronous_event_service_test",
   "grpc.framework.foundation._later_test",
-  "grpc.framework.foundation._logging_pool_test"
+  "grpc.framework.foundation._logging_pool_test",
```

Comment:
Hmm - sorry about not noticing this PR before I sent in #965. Now there're some collisions. :-/